README.md

```diff
@@ -2,145 +2,132 @@
-
-Using Vulcanize for the first time requires several steps be done in order to allow use of the software. The following instructions will offer a guide through the steps of the process:
-
-1. Fetching the project
-2. Installing dependencies
-3. Configuring shell environment
-4. Database setup
-5. Configuring synced Ethereum node integration
-6. Data syncing
-
-## Installation
-
-In order to fetch the project codebase for local use or modification, install it to your `GOPATH` via:
-
-`go get github.com/makerdao/vulcanizedb`
-`go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace`
-
-Once fetched, dependencies can be installed via `go get` or (the preferred method) at specific versions via `golang/dep`, the prototype golang pakcage manager. Installation instructions are [here](https://golang.github.io/dep/docs/installation.html).
-
-In order to install packages with `dep`, ensure you are in the project directory now within your `GOPATH` (default location is `~/go/src/github.com/makerdao/vulcanizedb/`) and run:
-
-`dep ensure`
-
-After `dep` finishes, dependencies should be installed within your `GOPATH` at the versions specified in `Gopkg.toml`.
-
-Lastly, ensure that `GOPATH` is defined in your shell. If necessary, `GOPATH` can be set in `~/.bashrc` or `~/.bash_profile`, depending upon your system. It can be additionally helpful to add `$GOPATH/bin` to your shell's `$PATH`.
-
-## Setting up the Database
+   
+### Getting the project
+Download the transformer codebase to your local local `GOPATH` via:
+`go get github.com/makerdao/vdb-mcd-transformers`
+
+## Usage
+As mentioned above this repository is a plugin for VulcanizeDB.  As such it cannot be run as a standalone executable,
+but instead is intended to be included as part of a VulcanizeDB core command. There are two VulcanizeDB core commands that 
+are required for events and storage slots to be transformed and persisted to the Postgres database.
+
+1. `headerSync` fetches raw Ethereum data and syncs it into VulcanizeDB's Postgres database where then it can be used for
+transformations. More information about the `headerSync` command can be found in the [VulcanizeDB repository](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/data-syncing.md#headersync).
+
+1. `execute` uses the raw Ethereum data that has been synced into Postgres and applies transformations to configured MCD
+contract data via [event](./transformers/events) and [storage](./transformers/storage) transformers. The VulcanizeDB
+repository includes a [general description](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/custom-transformers.md)
+about transformers.
+
+These core commands can be run via Docker images which is the preferred method, or they can be built and run via the
+command line interface. In either method, a postgres database will first need to be created:
 1. Install Postgres
 1. Create a superuser for yourself and make sure `psql --list` works without prompting for a password.
```

Comment:
Do you know if this is still necessary? Thinking we might be able to avoid it as long as the db user/pw are included in config

Reply:
It looks like we still need to create a supersuser for creating the `citext` extension which we're using to allow for case insensitivity for urn identifiers.
```
CREATE EXTENSION IF NOT EXISTS citext;
ERROR:  permission denied to create extension "citext"
HINT:  Must be superuser to create this extension.
```

I wonder if we should invest some time into figuring out a way around this, so we can remove the necessity for superusers.

Reply:
I might be focused too much on the part that says "make sure `psql --list` works without prompting for a password" - thinking maybe we can avoid that as long as the user/pw has the right permissions to run the migrations

Reply:
gotcha, yeah, that makes sense to me, does the change in f69b277 make sense?